src/bin/generate.rs

```diff
@@ -33,18 +41,26 @@ fn main() -> Result<(), Error> {
     let maybe_seed = match args.flag_seed_from {
         None => None,
         Some(path) => {
-            let mut hasher = Sha512Trunc256::new();
+            let mut hasher = Sha256::new();
```

Comment:
SHA-256 is an inferior cryptographic hash to truncated SHA-512. Could you explain why this change is desirable?

Reply:
I changed it back due to a previous change from you, where you said the upstream ed25519_dalek crate already used SHA256, so we should use it to avoid having more dependencies...

Reply:
I'm sorry, I might have mistyped that. Ed25519 uses SHA-512: https://github.com/dalek-cryptography/ed25519-dalek/blob/5870607aa770f8f08114fb8a8eae7e605b82b7c9/src/ed25519.rs#L26